scripts/ci/images/ci_verify_ci_image.sh

```diff
@@ -52,4 +52,6 @@ function pull_ci_image() {
 
 build_images::prepare_ci_build
 
+pull_ci_image
```

Comment:
This is just to explicitly pull the image in separate pull command. I found out that I have not done that.